.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,21 +4,18 @@ Thank you for contributing to the PrestaShop project!
 Please take the time to edit the "Answers" rows below with the necessary information.
 
 Check out our contribution guidelines to find out how to complete it:
-https://devdocs.prestashop.com/8/contribute/contribution-guidelines/#pull-requests
+https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/#pull-requests
 ------------------------------------------------------------------------------>
 
 | Questions         | Answers
 | ----------------- | -------------------------------------------------------
-| Branch?           | develop / 1.7.8.x
+| Branch?           | 8.0.x / develop / 1.7.8.x
 | Description?      | Please be specific when describing the PR. <br> Every detail helps: versions, browser/server configuration, specific module/theme, etc. Feel free to add more information below this table.
 | Type?             | bug fix / improvement / new feature / refacto
 | Category?         | FO / BO / CO / IN / WS / TE / LO / ME / PM
 | BC breaks?        | yes / no
 | Deprecations?     | yes / no
 | Fixed ticket?     | Fixes #{issue number here}.
-| Related PRs       | If theme, autoupgrade or other module change is needed, provide a link to related PRs here.
-| How to test?      | Please indicate how to best verify that this PR is correct.
-| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.
-
-
-<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->
+| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
+| Sponsor company   | Your company name goes here
+| How to test?      | Please indicate how to best verify that change works as expected.
```

Comment:
```suggestion
| How to test?      | It is required to indicate how to verify best that the change works as expected unless all the necessary steps are part of the related issue.
```

Reply:
I'd prefer to keep the message as short as possible. It's probably not necessary to state the "unless" part because people are already doing it.

How about: "Indicate how to verify that this change works as expected".

Reply:
And I think we should have `How to test` above other less important fields, maybe even after the `Description`

Reply:
@eternoendless good point

Reply:
@eternoendless what about moving "How to test"? If people don't want to create issues for every single PR, then this information is critical for committers and QA

Reply:
@eternoendless, apart from the wording I think it was discussed to move `How to test` a bit higher in the table to make it more visible